sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts

```diff
@@ -318,22 +320,22 @@ export const FastFrameTemplate = html<FastFrame>`
                             <span class="sample-control-icon"></span>
                             <span class="sample-control-text">Label</span>
                             <div class="sample-control-actions">
-                                <fast-button appearance="stealth"
+                                <fast-button appearance="stealth" aria-label="stealth"
```

Comment:
What's the purpose of this aria-label here? I don't think this is descriptive of the intent of the button. Should this say the appearance name, or should this say what this is doing?

Reply:
I agree it should say what the button is doing. In this case, the button here is only used to show what a button would look like when we change properties inside the fast frame control. My thought was to inform the user that its a stealth button instead of just a button. I could make it say "stealth button example" or "button example". But if we think just saying "button" is ok for this scenario, I can remove the aria-label.